app/helpers/template_helper.rb

```diff
@@ -35,4 +35,12 @@ def validate_closing_tags_exist(file_content)
 
     opening_tags.sort == closing_tags.sort
   end
+
+  def active_if(cond)
+    cond ? "active" : nil
+  end
+
+  def active_if_status(status)
+    status == "complete" ? "active" : nil
```

Comment:
it would be nice to use variables instead of strings here for "completed" and "active"